lang/cs/Org.Apache.REEF.Network/Elastic/Config/OperatorsParameters.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+using Org.Apache.REEF.Tang.Annotations;
+using Org.Apache.REEF.Utilities.Attributes;
+
+namespace Org.Apache.REEF.Network.Elastic.Config
+{
+    [Unstable("0.16", "Parameters may change")]
+    public class OperatorsParameters
+    {
+        [NamedParameter("Operator Name")]
+        public class OperatorType : Name<string>
```

Comment:
`public sealed class`?

Also, please don't package parameters as inner classes. Instead, put them into a namespace. The reason is that C# and Java represent inner classes differently and therefore can't read each other's Tang `Configuration` files if they use inner classes.